### ISSUE TITLE
Feedback on http://www.equal.vote/srvvsirv

### ISSUE THREAD

**Issue by wolftune** [nardo/Equal.Vote#19]
* In the IRV wording on the ballot, "goes to your next choice (if any)" should be more explicitly clear: "goes to your next choice (if you marked one and they have not yet been eliminated)". That would itself clear up the entire issue of understanding IRV.

**Comment by nardo** [comment]
It's up to date as of 30 minutes ago :-)

**Comment by wolftune** [comment]
I'm not going to get into editing the ballot image so left my comment for that. Will submit PR otherwise

**Comment by endolith** [comment]
It would probably be better if the SRV vs IRV image were "filled in" 

![](https://d3n8a8pro7vhmx.cloudfront.net/unifiedprimary/pages/126/attachments/original/1487232975/irv_srv_ballots.png?1487232975)

like this one:

![](https://d3n8a8pro7vhmx.cloudfront.net/unifiedprimary/pages/115/attachments/original/1482632716/SRV_sample_ballot.png?1482632716)

Showing the two ballots side by side for the same voter would show how IRV doesn't allow ties or degree of preference.  (Though if you kept names Jill vs Bernie etc you'd have to pick real life candidates, which would be bad for appearing partisan.  Maybe an animation that cycles through several ballots?  Meh.)

**Comment by wolftune** [comment]
In my experience, giving real candidate names leads people to ignore everything else and just focus on whether they imagine the system being good or bad for the candidates they prefer. That's a point for fictional names.

**Comment by endolith** [comment]
Yeah.  I think showing Hillary, Bernie, and Trump on the same ballot helps people realize that it eliminates the need for primaries, though.

**Comment by nardo** [comment]
We leave the ballots empty that use real candidate names, and A-F for ballots that are filled in on the site. I do think it'd be good to have a "ballot gallery" - where we can show what lots of different voters might do..., and we can label them all: the Bernie Bro, the #imwithher voter, the Greenie, the #maga fan, etc.
